### DISCUSSION TITLE
Using Cron Jobs to run triggers

### DISCUSSION THREAD

**Discussion by CrazyMoka** [quartznet/quartznet#1671] [Q&A] [answered]
Hello,
Does Quartz keep creating new cron jobs for each trigger with the same cron?
So If I want 10 jobs to run at 1pm and I create 10 triggers for those jobs does it make 10 crons?
Is it possible for it to use the same schedule for each trigger?

**Comment by lahma** [comment, marked as answer]
Ten triggers will be ten triggers. Multiple triggers can refer to same job,  but not the other way around. So you cannot share one trigger to multiple jobs.
